.zenodo.json

```diff
@@ -14,6 +14,11 @@
             "affiliation": "University of Wisconsin-Madison",
             "name": "Matthew Feickert",
             "orcid": "0000-0003-4124-7862"
+        },
+        {
+            "affiliation": "University of Liverpool",
+            "name": "Eduardo Rodrigues",
+            "orcid": "0000-XXX-YYY-ZZZ"
```

Comment:
@eduardo-rodrigues do you have an [ORCID](https://orcid.org/)? I didn't see if on your profile after a very quick check.

Reply:
Hi. Yep, see for example https://github.com/scikit-hep/particle/blob/master/.zenodo.json or similarly in the scikit-hep metapackage.